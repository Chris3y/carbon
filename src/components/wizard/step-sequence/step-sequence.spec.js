import React from 'react';
import { shallow } from 'enzyme';
import StepSequence from './step-sequence';

describe('<StepSequence />', () => {
  let stepSequence;

  beforeAll(() => {
    stepSequence = shallow(
      <StepSequence
        current='1'
        steps={ [
          { label: 'foo', state: 'complete' },
          { label: 'bar', state: 'current' },
          { label: 'baz', state: 'incomplete' }
        ] }
      />
    );
  });

  test('basic render', () => {
    expect(stepSequence).toMatchSnapshot();
  });
});
