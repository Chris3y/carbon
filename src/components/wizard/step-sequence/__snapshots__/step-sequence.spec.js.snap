// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StepSequence /> basic render 1`] = `
<ol
  className="carbon-step-sequence"
>
  <StepSequenceItem
    key="foo0"
    label="foo"
    state="complete"
    stepNumber={1}
  />
  <StepSequenceItem
    key="bar1"
    label="bar"
    state="current"
    stepNumber={2}
  />
  <StepSequenceItem
    key="baz2"
    label="baz"
    state="incomplete"
    stepNumber={3}
  />
</ol>
`;
